sw/airborne/modules/computer_vision/viewvideo.c

```diff
@@ -181,11 +198,12 @@ void *computervision_thread_main(void *data)
             // leave for loop
             break;
           } else {
-            printf("file exists\n");
+            //printf("file exists\n");
           }
         }
       }
-      computer_vision_thread_command = 1;
+      if (computer_vision_thread_command == 2)
+        computer_vision_thread_command = 1;
```

Comment:
this variable is not thread safe anymore: before only the module could instruct the thread to close, now the thread also writes this variable so it is possible that both write it at the same time. This change is not a fix, but reduces the critical time from jpeg-compression time to a single if statement.